tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts

```diff
@@ -218,6 +218,55 @@ describe('SDC School Collection View', () => {
       });
     });
 
+    it('verifies special filters for ELL tab', () => {
+      cy.intercept(Cypress.env('interceptors').collection_students_pagination).as('pagination');
+      cy.visit('/');
+      cy.get(selectors.dashboard.dataCollectionsTile).click();
+      cy.get(selectors.dataCollectionsLanding.continue).contains('Continue').click();
+      cy.get('button[value="English Language Learning"]').click();
```

Comment:
'button[value="English Language Learning"]' can go in the selectors file

Reply:
Hmm this is the pattern used in the other step 3 tests - I can put it in the selectors